VAMobile/e2e/tests/VeteranStatusCard.e2e.ts

```diff
@@ -202,5 +202,5 @@ describe('Veteran Status Card', () => {
         },
       ],
     )
-  })
+  })*/
```

Comment:
@rbontrager did you intend to leave several of these commented out?

Reply:
Nope. Fixed